src/rules/checkIndentation.js

```diff
@@ -17,6 +31,18 @@ export default iterateJsdoc(({
 }, {
   iterateAllJsdocs: true,
   meta: {
+    schema: [{
+      additionalProperties: false,
+      properties: {
+        excludeTags: {
+          items: {
+            type: 'string',
```

Comment:
@ahwayakchih 
I forgot this thing, can you add this in a follow up pr? Validating string passed to regexp looks like a good idea.   
```suggestion
            type: 'string',
            pattern: '^[a-zA-Z]+$',
```

Reply:
@golopot no problem. I added it to #391. To keep things super clean and tidy, it should be a separate PR, but the change is so tiny, i assumed it will be ok. Let me know if you want it separated.